<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuscleViewer</class>
 <widget class="QWidget" name="MuscleViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>911</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MuscleViewer</string>
  </property>
  <widget class="QPushButton" name="loadFrames">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>880</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Load Frames</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveTracks">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>880</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Save Data</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="exampleTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>861</height>
    </rect>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QScrollBar" name="frameScrollBar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>860</y>
     <width>881</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="GraphicsView" name="frameView">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>881</width>
     <height>841</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>890</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="play1X">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>890</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play2X">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>890</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playStop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>890</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="GraphicsView" name="frameHist">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTracks">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>880</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
